Project.toml

```diff
@@ -14,6 +14,7 @@ Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Formatting = "59287772-0a20-5a39-b81b-1366585eb4c0"
 IteratorInterfaceExtensions = "82899510-4779-5014-852e-03e436cf321d"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MLJModelInterface = "e80e1ace-859a-464e-9ed9-23947d8ae3ea"
```

Comment:
Can you add a MLJModelInterface to the `[compat]` section as well? Otherwise the merge bot in the Julia registry will complain.

Additionally, it would be good to bump the version number to `4.2.0` as well when finishing up the PR :slightly_smiling_face:

Reply:
Sounds good.